src/CoreBundle/Session/ChameleonSessionManager.php

```diff
@@ -142,6 +146,9 @@ public function boot()
 
         $session = new \TPKgCmsSession($sessionStorage);
 
+        $cookieName = "PHPSESSID" . $this->getCookieNameSuffix();
```

Comment:
This is a bit of a nitpick, but `PHPSESSID` should probably be moved into a constant. Sadly, PHP doesn't seem to have a built-in one for it.

Reply:
done